src/hotspot/share/gc/shenandoah/shenandoahHeapRegion.hpp

```diff
@@ -376,7 +382,9 @@ class ShenandoahHeapRegion {
 
   void print_on(outputStream* st) const;
 
-  void recycle();
+  void recycle_under_lock();
```

Comment:
Should be `try_recycle_under_lock` for consistency.

Reply:
Thanks! I have renamed it for consistency.